Packs/ApiModules/Scripts/TAXII2ApiModule/TAXII2ApiModule.py

```diff
@@ -940,10 +1013,10 @@ def get_entity_b_type_and_value(related_obj: str, id_to_object: dict[str, dict[s
         """
         indicator_obj = id_to_object.get(related_obj, {})
         entity_b_value = indicator_obj.get('name', '')
-        entity_b_obj_type = STIX_2_TYPES_TO_CORTEX_TYPES.get(indicator_obj.get('type', ''),
-                                                             STIX2XSOARParser.get_ioc_type(related_obj, id_to_object))
+        entity_b_obj_type = STIX_2_TYPES_TO_CORTEX_TYPES.get(
+            indicator_obj.get('type', ''), STIX2XSOARParser.get_ioc_type(related_obj, id_to_object))
         if indicator_obj.get('type') == "indicator":
-            entity_b_value = STIX2XSOARParser.get_ioc_value(related_obj, id_to_object)
+            entity_b_value = STIX2XSOARParser.get_single_pattern_value(id_to_object.get(related_obj, {}).get('pattern', ''))
```

Comment:
What if the value is `None`?

Reply:
That was an option before my changes too, but in any case, an indicator object must have a pattern so it's not really a concern.